docs/concepts.rst

```diff
@@ -81,33 +81,38 @@ Modes of operation
 Standalone mode
 ---------------
 
-In standalone mode, both the KSQL client and server components are
-co-located on the same machine, in the same JVM, and are started
-together. This makes standalone mode very convenient for local
-development and testing.
-
-.. image:: img/standalone-mode.png
+In standalone mode the KSQL server is started without an http listener.
```

Comment:
Perhaps:

> In certain deployment scenarios you may want to prevent interactive use of a KSQL cluster. For example, you want to allow a team of users to develop and test their queries on a shared KSQL cluster. But when putting those queries to production you prefer to lock-down access to KSQL servers, version-control the exact queries and storing them in a `.sql` file, and prevent users from interacting directly with the cluster.
>
> KSQL supports such locked-down deployment scenarios: you can configure servers to run a pre-defined script (`.sql` file) via the `--queries-file` command line argument or the `ksql.queries.file` setting in the KSQL configuration file. A server running such a pre-defined script will automatically disable its REST endpoint and thus disable interactive use.

Reply:
Thanks @miguno - i'll add that in

Reply:
Yer, what Martin says is definitely better ;)